app/components/bubbles.js

```diff
@@ -196,32 +194,30 @@ export class Bubbles extends Component {
         if (inData[idx].updating) return;
         inData[idx].updating = true;
         fetch(`${url}/api/v1/aggregates/present?register_ids=${point.id}`, { headers: prepareHeaders() })
-          .then(getJson)
-          .then(json => {
-            inData[idx].value = Math.floor(Math.abs(json.power_milliwatt)) || 0;
-            inData[idx].seeded = true;
-            inData[idx].updating = false;
-          })
-          .catch(error => {
-            inData[idx].updating = false;
-            console.log(error);
-          });
+        .then(getJson)
```

Comment:
:(
Editor connected to skynet in background. Now it have it's own will.
I'll fix it right now.